velox/functions/lib/DateTimeImpl.h

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
Consider the name `DateTimeDiff.h` if the file is mainly about timestamp differences.

Reply:
This class will add other common methods in the future, such as addTimestamp.